include/TPP/Passes.td

```diff
@@ -241,6 +241,15 @@ def RewriteBatchMatmulToMatmul : Pass<"rewrite-batch-matmul-to-matmul",
   let dependentDialects = ["scf::SCFDialect", "linalg::LinalgDialect"];
 }
 
+def ConvertPackOptimization : Pass<"pack-optimization", "func::FuncOp"> {
+  let summary = "Optimize certain pack patterns into faster implementation";
+  let description =
+      [{Optimize certain pack patterns into faster implementation.}];
```

Comment:
can you specify which types or patterns are you optimizing for? "certain" is a bit too generic.